source/Camera/Basic/CaptureHDR/CaptureHDR.cpp

```diff
@@ -18,7 +18,7 @@ int main()
 
         std::cout << "Configuring settings" << std::endl;
         Zivid::Settings settings;
-        for(const auto aperture : { 11.31, 5.66, 2.83 })
+        for(const auto aperture : { 9.57, 4.76, 2.59 })
```

Comment:
Just double checking - this has been tested on a 'regular' scene with Zivid One and Zivid Two cameras?

Reply:
Tested with some consumer goods, pack of cigarettes, small metallic cylinder-thingi, toy robot. A bit of everything.